boosty/api/api.py

```diff
@@ -1,4 +1,5 @@
-from pydantic import UUID4, BaseModel, conint
```

Comment:
The reason why you shouldn't use conint:
https://docs.pydantic.dev/latest/api/types/#pydantic.types.conint

Reply:
thanks!
I didn't know about this cool new feature, added in pydantic 2.4!